rubygem-hammer_cli_foreman_remote_execution/rubygem-hammer_cli_foreman_salt.spec

```diff
@@ -0,0 +1,75 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_foreman_remote_execution
+%global confdir hammer
+
+Summary: Foreman Remote Execution commands for Hammer CLI
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?dist}
+Group: Applications/System
+License: GPLv3
+URL: http://github.com/theforeman/hammer_cli_foreman_remote_execution
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+%if 0%{?fedora} > 18
+Requires: %{?scl_prefix_ruby}ruby(release)
+%else
+Requires: %{?scl_prefix_ruby}ruby(abi)
+%endif
+
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.1.3
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 0.5.0
```

Comment:
Please avoid pinning this tightly, it causes a test failure and more work each time a new major version of Hammer is shipped.  You should be OK to pin to `< 1.0.0`.  (See also cli-tasks which did this and had to continually update it.)